NYPL-discovery/discovery-front-end
# SCC-2568: Make catalog name configurable via environment variable

**What's this do?**
Makes all instances of "Shared Collection Catalog" reflect the `DISPLAY_TITLE` environment variable. Also, fixes the `npm test` script for the travis build, removing `source`. Removed and changed some code that was more complicated than necessary for tests.

**Why are we doing this? (w/ JIRA link if applicable)**
Prepare for launch 🚀
https://jira.nypl.org/browse/SCC-2568

**Do these changes have automated tests?**
Test suite as is, with some changes, reflect this update.

**How should this be QAed?**
Check
- 404 pages
- Page header
- Search results with no results
- Hold Confirmation page
- Landing page

**Dependencies for merging? Releasing to production?**
Need to update AWS config. Will probably still be "Shared Collection Catalog" for launch, change to "Research Catalog" on launch date.

**Has the application documentation been updated for these changes?**
Updated `env-sample` and `test.env`

**Did someone actually run this code to verify it works?**
I did
